### DISCUSSION TITLE
juicefs bench tool test result is not as expected

### DISCUSSION THREAD

**Discussion by LKR1996** [juicedata/juicefs#2334] [Q&A] [answered]
juicefs bench /mnt/jfs -p **4**
Cleaning kernel cache, may ask for root privilege...

  Write big blocks count: 4096 / 4096 [==============================================================]  done
   Read big blocks count: 4096 / 4096 [==============================================================]  done
Write small blocks count: 400 / 400 [==============================================================]  done
 Read small blocks count: 400 / 400 [==============================================================]  done
  Stat small files count: 400 / 400 [==============================================================]  done
Benchmark finished!
BlockSize: 1 MiB, BigFileSize: 1024 MiB, SmallFileSize: 128 KiB, SmallFileCount: 100, NumThreads: 4
Time used: 20.2 s, CPU: 135.7%, Memory: 657.1 MiB
+------------------+------------------+--------------+
|       ITEM       |       VALUE      |     COST     |
+------------------+------------------+--------------+
|    Read big file |    1259.89 MiB/s |  3.25 s/file |
|    Read big file |    1259.89 MiB/s |  3.25 s/file |
|        **Stat file |  15589.5 files/s | 0.26 ms/file |
|        Stat file |  15589.5 files/s | 0.26 ms/file |
|        Stat file |  15589.5 files/s | 0.26 ms/file |**
|   FUSE operation | 71298 operations |   0.97 ms/op |
|      Update meta |  1293 operations |   0.17 ms/op |
|       Put object |  1424 operations | 108.19 ms/op |
|       Get object |  1024 operations |  55.38 ms/op |
|    Delete object |     0 operations |   0.00 ms/op |
| Write into cache |  1304 operations |   2.04 ms/op |
|  Read from cache |   400 operations |   0.15 ms/op |****

**Comment by xiaogaozi** [comment, marked as answer]
This issue has been fixed in https://github.com/juicedata/juicefs/pull/2322
